index.js

```diff
@@ -124,4 +124,6 @@ WinstonCloudWatch.prototype.kthxbye = function(callback) {
 
 function stringify(o) { return JSON.stringify(o, null, '  '); }
 
+winston.transports.CloudWatch = WinstonCloudWatch;
```

Comment:
I think it's already being done by Winston.
If you try to run one of the examples and look into `winston` you should see `WinstonCloudwatch`.